packages/packages/api/src/medical/client/metriport.ts

```diff
@@ -41,35 +50,145 @@ export class MetriportMedicalApi {
   /**
    * Creates an org or updates one if it already exists
    *
-   * @param {Organization}    organization    - The org you want to create or update
-   *
+   * @param organization The org you want to create or update
    * @returns The created or updated org.
    */
   async createOrUpdateOrganization(organization: Organization): Promise<Organization> {
```

Comment:
Should we split the create/update for the org API while we're at it for consistency?

Reply:
I can, but I was hoping to merge this asap so I don't have to merge the suff from sandbox and can work on deploying the FHIR server.

Maybe add this as another task within the same ticket?

Reply:
^ sounds good to me

Reply:
https://github.com/metriport/metriport-internal/issues/438